src/libraries/System.Drawing.Common/src/System/Drawing/ImageAnimator.cs

```diff
@@ -35,6 +35,8 @@ namespace System.Drawing
     /// </summary>
     public sealed partial class ImageAnimator
     {
+        internal const int AnimationDelayMS = 40;
```

Comment:
I had to go back to find out where 40ms comes from -- I was reading the gif specs, etc. Looking at history, it's "some number a bit shorter than 50ms on the assumption that we won't show more than 20 frames a second". I wonder if that's worth a comment.

Reply:
Yep; good call. Sorry about the inconvenience there.

Reply:
```suggestion
        // We use a timer to apply an animation tick speeds of something a bit shorter than 50ms
        // such that if the requested frame rate is about 20 frames per second, we will rarely skip
        // a frame entirely. Sometimes we'll show a few more frames if available, but we will never
        // show more than 25 frames a second and that's OK.
        internal const int AnimationDelayMS = 40;
```